public/event-image-upload.js

```diff
@@ -0,0 +1,42 @@
+(function() {
+  var fileInput = document.getElementById('eventFileInput');
+  var fileErrorMessage = document.getElementById('fileErrorMessage');
+  var hiddenFileInput = document.getElementById('hiddenFileInput');
+  var spinner = document.getElementById('spinner');
+  var imagePreview = document.getElementById('imagePreview');
+
+  fileInput.onchange = function() {
```

Comment:
Do We want to let the user choose submit before uploading the file?

Reply:
I mean there is a spinner indicating that the uploading hasn't been done yet.
but i'll add a way to prevent them from submitting